### PR TITLE
Moved to actions, no doxygen necessary

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Comment by ladyada** [comment]
please remove travis

**Comment by dherrada** [comment]
@ladyada oh yeah, will do.

**Comment by dherrada** [comment]
hi @ladyada Should I be adding some .board.test.skips here?

**Comment by ladyada** [comment]
k for `StrandtestArduinoBLE.ino` and `StrandtestArduinoBLECallback.ino` add `.arduinoble.test.only` files

**Comment by dherrada** [comment]
Not sure why, but my computer wasn't registering that any of those changes needed to be made locally.

**Comment by ladyada** [comment]
oof the clang-formatting is not very helpful on this repo, can you rever the clangformat and remove the check? the actions part is good tho!

**Comment by dherrada** [comment]
@ladyada Will do

**Comment by dherrada** [comment]
@ladyada Ok, this is now passing

File: library.properties [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-version=1.4.0
+version=1.5.0
 author=Adafruit
 maintainer=Adafruit <info@adafruit.com>
 sentence=Arduino library for controlling single-wire-based LED pixels and strip.
 paragraph=Arduino library for controlling single-wire-based LED pixels and strip.
 category=Display
 url=https://github.com/adafruit/Adafruit_NeoPixel
 architectures=*
+depends=ArduinoBLE
```
Comment (ladyada) [inline library.properties:10]: remove this depends

**Pull request merged**